helpers/menu_tools.py

```diff
@@ -145,6 +145,24 @@ def add_menu_item(menu_id, button_text, command_name, index=None):
     return True
 
 
+def get_menu_item_index(menu_id, name):
```

Comment:
comment: this looks unused

Reply:
Good call. I never ended up using this function, and the function below it was originally used, but one of the later commits to this PR removes all usages of that function, so both functions are unused... Removed both!